scaffolds/java_actor_model/src/main/java/com/example/AkkaQuickstart.java

```diff
@@ -0,0 +1,57 @@
+package com.example;
+
+import akka.actor.ActorSystem;
+import akka.actor.AbstractActor;
+import akka.actor.ActorRef;
+import akka.actor.Props;
+
+import java.io.IOException;
+
+
+class MyActor extends AbstractActor {
+    public Receive createReceive() {
+        return receiveBuilder()
+        .match(String.class, p -> {
+            Thread.sleep(5000);
+            System.out.println("This is the String message. The message is: " + p);
+        })
+        .matchAny(p -> {
+            Thread.sleep(5000);
+            System.out.println("The class name of this message is: " + p.getClass().getName());
+        })
+        .build();
+    }
+}
+
+
+public class AkkaQuickstart {
+    public static void main(String[] args) throws InterruptedException{
+
+        final ActorSystem system = ActorSystem.create("Hello_Akka");
+
+        // Create an actor
+        ActorRef stringActor = system.actorOf(Props.create(MyActor.class), "String_actor");
+        // Send a String as a message for the actor
+        stringActor.tell("test message", ActorRef.noSender());
+
+        // crate 2 more actors and send different types of messages
+        ActorRef ref2 = system.actorOf(Props.create(MyActor.class), "Int_actor");
+        ref2.tell(1, ActorRef.noSender());
+        ActorRef ref3 = system.actorOf(Props.create(MyActor.class), "Double_actor");
+        ref3.tell(1.0, ActorRef.noSender());
```

Comment:
Could you also rename these actor variables?

Reply:
Yes, sure!